assignment-2/7s.md

```diff
@@ -1,2 +1,171 @@
+They have something called 'core metrics data'. The data is feed dynamically due constant experimentation.
+
+##### Staffing
+
+> Staffing refers to people in the organization and how they are recruited, selected, and trained to
+> perform their jobs. Leadership is included in this category, whether leaders are selected from the
+> outside or developed from within. Staffing addresses the question of how the organization
+> assures it has the right people to do the work.
+
+Spotify claims to have a very clear definition of what and how employees careers should develop in order to achieve the most important organization goals.[6]"Everyone in *Spotify tech* should have a way to grow their careers and expand the impact of their -work no matter what role they play".
+The Spotify technology career framework, consist on letting the employee transfer itself on as many roles and domain expertise as possible in order to gain experience and later evaluate their impact into the organization.
+*On recognition and promotions* Spotify observes constantly the impact someone has at a wider scope, and they have a formal ways to promote their employees
+*On setting the step for a new employee* Spotify gives 6 months to all new employees to get involved with their role and the organization.
+*On compensation* A formal promotion should lead to higher pay in a straightforward and transparent way.
+
+##### Skills
+
+> Skills are the special, distinctive competencies of the organization, both in the people and the
+> organization itself. What is the organization good at, and what special skills and competencies
+> are found its people?
+
+What is the organization good at?
+Spotify seems to be good at keeping their organization dynamic, focusing on their community rather to an hierarchical structure.
+
+##### Style
+
+> Style deals mainly with the leadership approach of the top managers in the organization. How do
+> the leaders interface with subordinates and others in the organization, and how do members
+> interact with each other? Style also incorporates the climate or culture of the organization, which
+> can be expressed in terms of job titles, space allocation, workplace architecture, artifacts, and
+> policies.
+
+How do the leaders interact with subordinates and others in the organization?
+According with Spotify [12]:
+- They have an Agile engineering culture, which means they like servent leaders rather than process masters.
+- Autonomy that every member team has over their work have evolve to a high rate percentage of team members satisfaction and they keep investing on it. 
+- Autonomy seems also to keep high motivation levels [12 and 13], teams are fast, and their minimize de dependency from and to, other teams.
+ 
+##### Shared Values
+
```

Comment:
This is probably the most important thing of the 7-Asses. What is the vision, what is the mission. What drives them. There mission is surely not 'keeping the organization a nice place'. They have high level statements and a plan on how to achieve this. Please elaborate on this.